src/main/java/com/drinchev/projectlabel/utils/UtilsFont.java

```diff
@@ -41,7 +42,15 @@ public static Font setAttributes(Font font, Float weight, Float size) {
     }
 
     public static JBFont getStatusBarItemFont() {
-        return JBFont.medium();
+        try {
+            // while the new UI is experimental, we have to carefully choose the font size
+            if (JBFont.class.getDeclaredMethod("smallOrNewUiMedium") != null) {
+                return Registry.is("ide.experimental.ui") ? JBFont.medium() : JBFont.small();
```

Comment:
@simschla I've added your suggestion here, as well as in determining the label's height for the two UI's on line 38.